discordpp/log.hh

```diff
@@ -15,12 +15,22 @@ using handleLog = std::function<void(std::ostream *)>;
 enum level { trace, debug, info, warning, error, fatal };
 inline level filter = warning;
 inline std::ostream *out = &std::cout;
+inline std::ostream* err = &std::cerr;
+inline std::ostream* heartbeat = &std::cout;
 
 inline void log(level l, handleLog msg) {
     if (l >= filter) {
         msg(out);
     }
 }
+inline void logerr(handleLog msg)
+{
+    msg(err);
+}
+inline void logheartbeat(handleLog msg)
```

Comment:
I specifically said not to touch the heartbeats?

Reply:
ok, I'll remove it